{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/common.py

```diff
@@ -131,7 +131,7 @@ class Common(Configuration):
 
     # Absolute filesystem path to the directory that will hold user-uploaded files.
     # Example: "/var/www/example.com/media/"
-    MEDIA_ROOT = os.path.join(BaseDir.BASE_DIR, 'media')
+    MEDIA_ROOT = values.Value(os.path.join(BaseDir.BASE_DIR, 'media'))
```

Comment:
Please change to [PathValue](https://django-configurations.readthedocs.org/en/stable/values/#configurations.values.PathValue)